.golangci.yml

```diff
@@ -9,7 +9,6 @@ run:
 linters:
   disable-all: true
   enable:
-    - depguard
```

Comment:
it works before?

Reply:
yea, [v2](https://github.com/openpeedeep/depguard/tree/v2) use analysis package